app/docs/0.13.x/admin-api.md

```diff
@@ -1257,64 +1258,8 @@ Attributes | Description
 ---:| ---
 `id`<br>*optional* | A filter on the list based on the `id` field.
 `name`<br>*optional* | A filter on the list based on the `name` field.
-`api_id`<br>*optional* | A filter on the list based on the `api_id` field.
-`consumer_id`<br>*optional* | A filter on the list based on the `consumer_id` field.
-`size`<br>*optional, default is __100__* | A limit on the number of objects to be returned.
-`offset`<br>*optional* | A cursor used for pagination. `offset` is an object identifier that defines a place in the list.
-
-#### Response
-
-```
-HTTP 200 OK
-```
-
-```json
-{
-    "total": 10,
-    "data": [
-      {
-          "id": "4d924084-1adb-40a5-c042-63b19db421d1",
-          "api_id": "5fd1z584-1adb-40a5-c042-63b19db49x21",
-          "name": "rate-limiting",
-          "config": {
-              "minute": 20,
-              "hour": 500
-          },
-          "enabled": true,
-          "created_at": 1422386534
-      },
-      {
-          "id": "3f924084-1adb-40a5-c042-63b19db421a2",
-          "api_id": "5fd1z584-1adb-40a5-c042-63b19db49x21",
-          "consumer_id": "a3dX2dh2-1adb-40a5-c042-63b19dbx83hF4",
-          "name": "rate-limiting",
-          "config": {
-              "minute": 300,
-              "hour": 20000
-          },
-          "enabled": true,
-          "created_at": 1422386585
-      }
-    ],
-    "next": "http://localhost:8001/plugins?size=2&offset=4d924084-1adb-40a5-c042-63b19db421d1"
-}
-```
-
----
-
-### List Plugins per API
-
-#### Endpoint
-
-<div class="endpoint get">/apis/{api name or id}/plugins/</div>
-
-#### Request Querystring Parameters
-
-Attributes | Description
----:| ---
-`id`<br>*optional* | A filter on the list based on the `id` field.
-`name`<br>*optional* | A filter on the list based on the `name` field.
-`api_id`<br>*optional* | A filter on the list based on the `api_id` field.
+`service_id`<br>*optional* | A filter on the list based on the `service_id` field.
+`route_id`<br>*optional* | A filter on the list based on the `route_id` field.
```

Comment:
it seems like we removed `consumer_id` here

Reply:
Hmm I see - you want to remove "deprecated" `/apis/.../` endpoints?

Also: all changes made to the sections of this doc should be also reflected in the YAML sidebar

Reply:
> Hmm I see - you want to remove "deprecated" /apis/.../ endpoints?

Indeed

> all changes made to the sections of this doc should be also reflected in the YAML sidebar

Done!